### ISSUE TITLE
No Save option for plots (for R code in markdown cell) 

### ISSUE THREAD

**Issue by sanketd11** [att/rcloud#1437]
For a markdown cell containing R code for any plot, the save option is not displayed for the plots.

![111save](https://cloud.githubusercontent.com/assets/9251570/7564717/2d716d40-f807-11e4-927f-a066c2fd57c3.PNG)

**Comment by gordonwoodhull** [comment]
Right, we would have to reimplement knitr or learn more about its internals in order to make this happen. As of 1.3 it is an entirely different mode of processing for R cells vs Markdown cells.

Perhaps if knitr is using the htmltools framework there is a customization point, but this is pure speculation.
